pkg/reconciler/service/resources/configuration.go

```diff
@@ -39,7 +39,7 @@ func MakeConfiguration(service *v1alpha1.Service) (*v1alpha1.Configuration, erro
 				serving.RouteLabelKey:   names.Route(service),
 				serving.ServiceLabelKey: service.Name,
 			}),
-			Annotations: service.GetAnnotations(),
+			Annotations: copyAnnotations(service.GetAnnotations()),
```

Comment:
I think this should rather be:

```golang
resources.FilterMap(service.GetAnnotations(), func(key string) bool {
  if annotationsToSkip[key] {
    return false
  }
  return true
})
```

Reply:
That's better, I've updated the PR. Thank you.

My understanding is that annotations get copied from parent to child resource so filtering annotations at Knative Service level should stop it propagating to subsequent child resources.